system-tests/projects/qwik-app/cypress-ct-bad-missing-value/definition.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
What is the missing value? Can we make this more minimal? It's kind of hard to see the actual problem